emacs.fnl

```diff
@@ -47,17 +49,8 @@
       (: edit-window :moveToScreen scr)
       (: windows :center-window-frame))))
 
-(fn run-emacs-fn
```

Comment:
This function already exists, I accidentally created a dup while rebasing master to develop